src/test/unit/marker-data.test.js

```diff
@@ -230,6 +231,22 @@ describe('deriveMarkersFromRawMarkerTable', function() {
       title: null,
     });
   });
+  it('should create a marker for the marker UserTiming', function() {
```

Comment:
`'should create a marker for the marker UserTiming'` is incorrect. It should be `'should create a marker for the marker CompositorScreenshot'`

Reply:
Oops copy/paste fail 😄